### DISCUSSION TITLE
KH.dat and Hydrostatics.dat files using WEC-Sim

### DISCUSSION THREAD

**Discussion by AlufaSam** [capytaine/capytaine#318] [General]
@mancellin, please, I need to run a time series simulation with WEC-Sim using Capytaine as my BEM software. To generate the .h5 file, I need the output from Capyatine (the .nc format I already generated) and files containing the Hydrostatics data (KH.data and Hydrostatics.dat). Please, how do I generate these .dat files? I have tried to import meshmagick, but I get an error (I used git to install meshmagick in the same folder as Capytaine,  but I still get this error).
![image](https://user-images.githubusercontent.com/105754974/228953004-80199645-1c20-4c46-ade8-a7d1e51a5647.png)

![image](https://user-images.githubusercontent.com/105754974/228953509-f1cb510a-e618-4558-a520-8ae28f97a39f.png)

**Comment by mancellin** [comment]
Your error seems to be a problem with the installation of `meshmagick`.
Did you install it in your Python environment with
```
pip install https://github.com/LHEEA/meshmagick/archive/master.zip
```
as recommended on their Github page?


In the next version of Capytaine, a new function by @dav-og will be included to export hydrostatics data in this format (cf #285). Unfortunately I won't have time to finish the next release before a few weeks.

**Reply by AlufaSam** [reply]
I earlier used git cmd to install it to the folder containing capytaine. Now I have used "pip install ..." with anaconda to install it in the same environment as capytaine, and it works.
Thanks.
